### DISCUSSION TITLE
Complexity of stories, HOC components?

### DISCUSSION THREAD

**Discussion by flexa-david** [clauderic/dnd-kit#764] [General]
I've been trying to implement the vertical sortability to my own project components by making use of the story example, however the examples are very complex and have a lot of styling critical to achieving the desired functionality. Is the task of making things sortable inherently complex even utilising this library, or is there some high order components I should be using that make this trivial?

**Comment by RemyMachado** [comment]
I think it's a great idea. There is a need for HOCs and more hooks. We find ourselves diggind through the source-code to understand how to replicate examples.

For instance, as simple as "Getting a `draggable` to stay in place when dropped" -> Needs a HOC that handles coordinates on `onDragEnd`.
